jobs/config.json

```diff
@@ -1048,6 +1060,7 @@
       "--env=CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1",
       "--extract=ci/k8s-stable1",
       "--gcp-node-image=gci",
+      "--gcp-project-type=ingress-project",
```

Comment:
`     "_comment": "AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT.",`

I guess add the flag to https://github.com/kubernetes/test-infra/blob/master/experiment/test_config.yaml#L944-L947 and run `bazel run //experiment:generate_tests -- --yaml-config-path=experiment/test_config.yaml --json-config-path=jobs/config.json  --prow-config-path=prow/config.yaml && bazel run //jobs:config_sort`

Reply:
Thanks for the command, updated now :)